### PR TITLE
Update To Add java.time Immutable Classes To JdkConfiguration.

### PR DESCRIPTION
**Pull request by bharatbhasin**

### CODE REVIEW

File: src/main/java/org/mutabilitydetector/config/JdkConfiguration.java [hunk 1]
```diff
@@ -41,24 +55,52 @@
...
+		hardcodeAsDefinitelyImmutable(Short.class);
+		hardcodeAsDefinitelyImmutable(Integer.class);
+		hardcodeAsDefinitelyImmutable(Long.class);
+		hardcodeAsDefinitelyImmutable(Float.class);
+		hardcodeAsDefinitelyImmutable(Double.class);
+		hardcodeAsDefinitelyImmutable(BigDecimal.class);
+		hardcodeAsDefinitelyImmutable(BigInteger.class);
+		hardcodeAsDefinitelyImmutable(Class.class);
+		hardcodeAsDefinitelyImmutable(URI.class);
+		hardcodeAsDefinitelyImmutable(Duration.class);
```
Comment (remal) [inline src/main/java/org/mutabilitydetector/config/JdkConfiguration.java]:
```suggestion
		hardcodeAsDefinitelyImmutable("java.time.Duration");
```

And the same change for other lines.

Comment (Grundlefleck) [reply src/main/java/org/mutabilitydetector/config/JdkConfiguration.java]: @remal what leads you to believe they need to be quoted as fully qualified class names?

Comment (bharatbhasin) [reply src/main/java/org/mutabilitydetector/config/JdkConfiguration.java]: Hey Remal,
There is an overridden version hardcodeAsDefinitelyImmutable(Class<?> immutableClass), which is being used the JdkConfiguration class.

**Comment by Grundlefleck** [comment]
No response, I think not having a quoted class name is fine. Shall find out if I'm wrong.

**Comment by Grundlefleck** [comment]
@bharatbhasin thank you for your contribution, I appreciate it.

**Pull request merged**